### PR TITLE
Blog app - add controllers

### PR DESCRIPTION
**Pull request by selma-belhadj**
## What does this PR do?
This PR is raised to create all controllers and views for the Blog app by adding simple placeholders for the views with plain HTML.

## Description of the completed tasks
- Created for each URL :
  - A route.
  - An action in the correct controller.
  - A view file.
- Included only placeholders written in plain HTML in the view files.
- Added code to handle only the 4 required URLs.
## How should it be tested
- clone the repository
`git clone https://github.com/selma-belhadj/BlogApp.git`
- navigate to the folder
`cd BlogApp`
- checkout the feature
`git checkout model-specs`
- Install packages
`bundle install`
- Setup database
`rails db:setup`
- Run migration
`rails db:migrate`
- Run tests
`rspec`
- Run the app
 `rails s`

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1>Posts List</h1>
+
+<p>Here is a list of all posts for a given user bas ed on the given user_id</p>
```
Comment (Singh212dab) [inline app/views/posts/index.html.erb]: p>Here is the list of all posts for a given user based on the given user_id</p>

Comment (Singh212dab) [reply app/views/posts/index.html.erb]: corrected the statement

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (selma-belhadj) [reply app/views/posts/index.html.erb]: @Singh212dab You mean based !!

**Review by dasileker — APPROVED** [review]
### **Hi @selma-belhadj,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**